File: Utt2Database/ConfigProp.vb
Imports System
Imports System.Collections.Generic
Imports Microsoft.VisualBasic

Namespace Utt2Database
    Partial Public Class ConfigProp
        Public Property Key As String

        Public Property Data As String

        Public Property [Private] As Boolean

        Public Overrides Function ToString() As String
            Return Me.Key
        End Function
    End Class
End Namespace

File: src/IndexerSharp/Utt2Database/ConfigProp.cs
namespace Naomai.UTT.Indexer.Utt2Database;
public partial class ConfigProp
{
    public string Key { get; set; } = null!;
    public string Data { get; set; } = null!;
    public bool IsPrivate { get; set; }
    public override string ToString()
    {
        return Key;
    }
}
